skyportal/handlers/api/allocation.py

```diff
@@ -222,3 +228,139 @@ def delete(self, allocation_id):
         self.verify_and_commit()
         self.push_all(action='skyportal/REFRESH_ALLOCATIONS')
         return self.success()
+
+
+class AllocationAnalysisHandler(BaseHandler):
```

Comment:
Maybe `AllocationReportHandler` makes it more distinct from "analysis" which is a whole other thing?

Reply:
Yes good point.